mis_builder_cash_flow/views/mis_cash_flow_forecast_line_views.xml

```diff
@@ -0,0 +1,96 @@
+        <field name="model">mis.cash_flow.forecast_line</field>
+        <field name="arch" type="xml">
+            <form string="MIS Cash Flow Forecast Line">
+                <sheet>
+                    <group col="2">
+                        <!-- pylint:disable=duplicate-xml-fields -->
+                        <field name="company_id" groups="base.group_multi_company" />
+                        <field name="company_id" invisible="1" />
+                        <field name="date" />
+                        <field name="partner_id" />
+                        <field
+                            name="account_id"
+                            domain="[('company_id', '=', company_id)]"
+                        />
+                        <field name="name" />
+                        <field name="balance" />
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="mis_cash_flow_forecast_line_view_search">
+        <field name="name">mis.cash_flow.forecast_line.search</field>
+        <field name="model">mis.cash_flow.forecast_line</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name" />
+                <field name="date" />
+                <separator />
+                <field name="partner_id" />
+                <field name="account_id" />
+                <field name="company_id" groups="base.group_multi_company" />
+                <group>
+                    <filter
+                        string="Account"
+                        name="group_by_account_id"
+                        context="{'group_by':'account_id'}"
+                    />
+                    <filter
+                        string="Partner"
+                        name="group_by_partner_id"
+                        context="{'group_by':'partner_id'}"
+                    />
+                    <filter
+                        string="Date"
+                        name="group_by_date"
+                        context="{'group_by':'date'}"
+                    />
+                    <filter
+                        string="Company"
+                        name="group_by_company_id"
+                        context="{'group_by':'company_id'}"
+                        groups="base.group_multi_company"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="mis_cash_flow_forecast_line_view_tree">
+        <field name="name">mis.cash_flow.forecast_line.tree</field>
+        <field name="model">mis.cash_flow.forecast_line</field>
+        <field name="arch" type="xml">
+            <tree editable="top">
+                <!-- pylint:disable=duplicate-xml-fields -->
+                <field name="company_id" groups="base.group_multi_company" />
+                <field name="company_id" invisible="1" />
```

Comment:
It seems to me that invisible in tree is no longer functional. You must use column_invisible.

Reply:
Yeah, that's true. cc @cormaza